README.md

```diff
@@ -118,6 +118,8 @@ This implementation does away with the string concatenation (and resulting strin
 | `UUID#fromString(String)`          |  2,613,730 UUIDs/second |  ± 25,278 UUIDs/second |
 | `FastUUID#parseUUID(String)`       | 16,796,302 UUIDs/second | ± 191,695 UUIDs/second |
```

Comment:
Method signature is wrong here, btw @jchambers. I'll fix it, if we're keeping the `parseUUID(CharSequence)` signature.